lib/mapper.js

```diff
@@ -164,14 +167,16 @@ function createBaseTrackProperties(track, settings) {
   if (settings.prefixProperties) {
     properties = prefix(track.event(), properties);
   }
-
-  return extend(
-    properties,
-    {
+  // Only apply userAgent property on `visited site` event
+  if (track.event() === 'visited site') {
```

Comment:
What should be the value in those instances?

Should I change this then to something along the lines of:
```
properties._ua = track.event() || track.proxy('context.userAgent') || 'DEFAULT_VALUE'
```